amethyst_renderer/src/pipe/mod.rs

```diff
@@ -98,11 +100,18 @@ impl Pipeline {
     pub fn enabled_stages(&self) -> Stages {
         Stages(self.stages.iter().filter(|s| s.is_enabled()))
     }
-
+	/// Get the mutable stages
+    pub fn stages_mut(&mut self)->&mut Vec<Stage>{
```

Comment:
Any reason why you made the field public **and** added a getter? Additionally, is every possible safe modification of the stages vector valid API usage?